Huizenjacht/src/main/resources/application.properties

```diff
@@ -1,3 +0,0 @@
-server.port=9991
-logging.level.be.pxl=debug
-server.error.include-binding-errors=always
```

Comment:
Ipv de volgende properties te gebruiken welke een zeer uitgebreide array van validation checks teruggeeft
server.error.include-message=always
server.error.include-binding-errors=always

kan je ook werken met een globale excpetion handler werken om de validation errors af te handelen
Zie GlobalExceptionHandler p62-63 in de cursus